ports/stm/supervisor/usb.c

```diff
@@ -115,17 +119,27 @@ void init_usb_hardware(void) {
     #endif
 
     #if CPY_STM32H7
+    RCC_PeriphCLKInitTypeDef PeriphClkInitStruct = {0};
+
+    /** Initializes the peripherals clock */
+    PeriphClkInitStruct.PeriphClockSelection = RCC_PERIPHCLK_USB;
+    PeriphClkInitStruct.UsbClockSelection = RCC_USBCLKSOURCE_HSI48;
+    HAL_RCCEx_PeriphCLKConfig(&PeriphClkInitStruct);
+
     HAL_PWREx_EnableUSBVoltageDetector();
-    __HAL_RCC_USB2_OTG_FS_CLK_ENABLE();
+    __HAL_RCC_USB_OTG_HS_CLK_ENABLE();
```

Comment:
Do some H7 have two USB peripherals? The board should probably configure which to use.